fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/AbstractResource.java

```diff
@@ -97,7 +98,12 @@
     @Optional
     protected Supplier<String> pidMinter;
 
-    protected UniqueValueSupplier defaultPidMinter = new DefaultPathMinter();
+    // TODO: Allow configuration properties to override to length and count
+    final int desiredLength = HierarchicalIdentifierSupplier.DEFAULT_LENGTH;
+    final int desiredCount = HierarchicalIdentifierSupplier.DEFAULT_COUNT;
+
+    protected UniqueValueSupplier defaultPidMinter =
+            new ConfigurableHierarchicalSupplier(desiredLength, desiredCount);
```

Comment:
If you directly instantiate the `ConfigurableHierarchicalSupplier` above, then this is not required and could be removed.